.github/workflows/ci.yaml

```diff
@@ -636,6 +636,53 @@ jobs:
           name: ecosystem-result
           path: ecosystem-result
 
+  fuzz-redknot:
+    name: "Fuzz for new red-knot panics"
+    runs-on: depot-ubuntu-22.04-16
+    needs:
+      - cargo-test-linux
+      - determine_changes
+    # Only runs on pull requests, since that is the only we way we can find the base version for comparison.
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'no-test') && github.event_name == 'pull_request' && needs.determine_changes.outputs.red_knot == 'true' }}
```

Comment:
Yeah, and it's most useful to run it on a PR. We also don't run the py-fuzzer on parser on `main`.